client/js/components/header/search.js

```diff
@@ -0,0 +1,43 @@
+import showHide from './../show-hide';
+import { KEYS } from './../../util';
+
+const headerSearch = (el) => {
+  const headerSearchInput = el.querySelector('.js-search-input');
+  const searchToggle = showHide({el: el});
+
+  const init = () => {
+    handleEvents();
+  };
+
+  const handleEvents = () => {
+    el.addEventListener('submit', (event) => {
+      if (searchToggle.getActive() && headerSearchInput.value.trim().length) return;
```

Comment:
I'm starting to agree that it might just be most straightforward to take out the search for now until we're slightly further down the line with how it's going to be implemented. Will do so in a commit – we can always cherry-pick it back in if it turns out to be close.